### DISCUSSION TITLE
无语瞌睡猫表情包

### DISCUSSION THREAD

**Discussion by xmexg** [NapNeko/NapCatQQ#589] [General]
![logo无语瞌睡猫](https://github.com/user-attachments/assets/ee176efb-d4ca-467a-8cd2-172ab9547080)

**Comment by xmexg** [comment]
![超级无语瞌睡猫](https://github.com/user-attachments/assets/252e5451-7272-45d3-994d-97100ec41ddb)

**Reply by VincentZyu233** [reply]
可爱
